plugins/modules/eos_ntp_global.py

```diff
@@ -0,0 +1,1000 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2021 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+"""
+The module file for eos_ntp_global
+"""
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+---
+module: eos_ntp_global
+short_description: Manages ntp resource module
+description: This module configures and manages the attributes of  ntp on Arista
+  EOS platforms.
+version_added: 3.1.0
+author: Gomathi Selvi Srinivasan (@GomathiselviS)
+notes:
+- Tested against Arista EOS 4.24.60M
+- This module works with connection C(network_cli). See the L(EOS Platform Options,eos_platform_options).
```

Comment:
I don't think the L() works here because ansible/ansible lives in a separate repository. It likely needs the full URL. @KB-perByte can you confirm?

Reply:
Yep, we need to use U(full_url) everywhere in place of L() 
Eg: See U(https://docs.ansible.com/ansible/latest/network/user_guide/platform_eos.html)

Reply:
ref PR: https://github.com/ansible-collections/cisco.ios/pull/368